packages/map-template/src/components/Map/MapboxMap/MapboxMap.scss

```diff
@@ -0,0 +1,31 @@
+.mapboxgl-control-container {
+    .mapboxgl-ctrl {
+        &-group {
+            margin: 10px 12px 10px 0 !important;
+            border-radius: 8px;
+            box-shadow: 0 0 0 2px #e5e7eb !important;
+
+            button {
+                padding: 24px; // UX design
+            }
+        }
+
+        &-zoom-in {
+            background-image: url('../../../assets/plus.svg');
+            background-repeat: no-repeat;
+            background-size: 20px 20px;
+            background-position: center center;
+        }
+
+        &-zoom-out {
+            background-image: url('../../../assets/minus.svg');
+            background-repeat: no-repeat;
+            background-size: 20px 20px;
+            background-position: center center;
+        }
+
+        &-icon {
+            background-image: none;
+        }
+    }
+}
```

Comment:
For consistency reasons, could we use midt variables here? 🙏🏻
See examples in the other `scss` files @matbmapspeople